table/server/underdb/glue/src/main/java/alluxio/table/under/glue/GlueDatabase.java

```diff
@@ -305,6 +308,7 @@ public UdbTable getTable(String tableName) throws IOException {
     // Glue doesn't support column statistics infomation
     Map<String, List<ColumnStatisticsInfo>> statsMap = new HashMap<>();
```

Comment:
Instead of creating a new object, we can use `Collections.emptyMap()`

Reply:
Changed.